R/cluster.R

```diff
@@ -220,10 +241,11 @@ makeCluster <-
         fill = TRUE
       )
 
-
       clusterNodeMismatchWarning <-
-        paste0("There is a mismatched between the projected cluster %s ",
-               "nodes min/max '%s'/'%s' and the existing cluster %s nodes '%s'")
+        paste0(
+          "There is a mismatched between the projected cluster %s ",
+          "nodes min/max '%s'/'%s' and the existing cluster %s nodes '%s'"
```

Comment:
Ok... And what can the user do about it? Is there a standard action or set of actions they can take? Is this going to get them into a bad state?

Also, may want to precede the message "WARNING: ..."

Reply:
1) Adding a comment about using `resizeCluster` function. This will not get them into a bad state.. Unless you consider having less workers is a bad state. However, you'll be able to run your job fine. That's why I consider it a warning instead of an error.  

2) It'll precede warning in the warning function